bundles/org.openhab.binding.tapocontrol/README.md

```diff
@@ -25,7 +25,7 @@ The following Tapo-Devices are supported. For precise channel-description look a
 | Smart Temperature Sensor (RF)      | T310        | Temperature and Humidity Sensor              |
 |                                    | T315        | Temperature and Humidity Sensor with Display |
 | Smart Switch (RF)                  | S210        | Wall switch with 1 contact                    |
-| Smart Switch (RF)                  | S220        | Wall switch with 2 contacts                  | 
+| Smart Switch (RF)                  | S220        | Wall switch with 2 contacts                   |
```

Comment:
```suggestion
| Smart Switch (RF)                  | S220        | Wall switch with 2 contacts                  |
```

(it's the line above that has wrong indentation)

Reply:
Can’t fix this on my phone without creating new issues. I’ll probably create a follow up PR that will fix all table markup. (And adds a ci check ) please merge this as is, rendered layout should not have been negatively impacted.